parity/cli/mod.rs

```diff
@@ -438,6 +438,11 @@ usage! {
 			"--port=[PORT]",
 			"Override the port on which the node should listen.",
 
+			ARG arg_interface: (String)  = "all", or |c: &Config| c.network.as_ref()?.interface.clone(),
+			"--interface=[IP]",
+			"Specify the hostname portion on which the node should listen, IP should be an interface's IP address, or all (all interfaces) or local.",
```

Comment:
Yeah, I thought this wording is weird, too. Tried to be close to the other `--interface`-settings (jsonrpc, api, etc) but happily take yours, too..

> Also: if I have 5 NICs and want to listen on two of them, will `--interface=192.168.0.1, 172.16.0.5` work?

No. First off because the parser wouldn't be able to understand it, and secondly, because of the way the code is set up, it only knows about one socket - this would require multiple sockets afaik. 

Considering that all other instances of `--*-interface` only support a single bind, too, I don't think refactoring to support multiple interfaces isn't really worth it at the moment (and even if, shouldn't be part of this PR IMHO).

Reply:
Thank you for clarifying. If we only support a single interface though, what is the meaning of `--interfaces=all`? I don't think that should be a valid value.

Reply:
`all` is the equivalent of `0.0.0.0` (but easier to write), which if bound to means "all interfaces" -- as technically, `bind` doesn't actually go to an interface, but means listening for a certain IP and in that context `0.0.0.0` means _any_ IP.

I should have said, we only support a single socket, rather than a single interface, as you are right, we can express multiple interface within the same socket through `0.0.0.0`. 

What I meant to say, was that, we can bind to a specific subset: we can bind to one or all.